src/select-list-view.js

```diff
@@ -37,6 +37,14 @@ module.exports = class SelectListView {
     this.refs.queryEditor.setText('')
   }
 
+  setQueryText (value, isSelect = false) {
```

Comment:
@as-cii sorry, it's still not clear for me why you want it to be as object. For both variant developer will have to know the signature of the method:
`setQuery("test",true)`
`setQuery("test",{selectText:true})`
But in second case he will have to know the innner representation of the param and expose it outside the implementation and additionally calling setQuery method is much longer in second case.
Could you give me some details on this?